shared/rakefiles/entrypoint.rake

```diff
@@ -70,21 +72,15 @@ task :configure_serviceaccount => [:set_vars] do
   sh "#{@exekube_cmd} rake configure_serviceaccount"
 end
 
-desc "[ADVANCED] Fetch kubectl credentials (gcloud auth login)"
-task :configure_kubectl => [:set_vars] do
-  sh "#{@exekube_cmd} rake configure_kubectl"
-end
+desc "Get the credentials needed to create all the resources inside the project"
+task :project_init => [:set_vars, :configure_current_project, :configure_serviceaccount]
```

Comment:
Is it expected that this task has no contents?

Reply:
Yes, it is expected. This task "executes" the list of dependencies.